setup.py

```diff
@@ -87,7 +87,6 @@ def run(self):
     },
     include_package_data=True,
     install_requires=_REQUIREMENTS,
-    setup_requires=['setuptools_scm', 'numpy', 'cython'],
```

Comment:
`setup_requires` [is deprecated](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html?highlight=setup_requires#package-requirement) in favor of the pyproject.toml `requires`.